source/03-components/button/_button.scss

```diff
@@ -18,21 +18,28 @@
   }
 }
 
+// This button class is used for Drupal tabs.
+.button--base {
+  @include button(
+    $color-background: gesso-color(button, base, background),
```

Comment:
Is grey what USWDS uses? It seems like an odd default to me, because I typically interpret grey buttons to be disabled.

Reply:
Yep, USWDS uses gray for their Base button variant. I’m not a fan of the “Base” name, but I’m not sure what to call it.

Reply:
Yep, USWDS uses gray for their Base button variant. I’m not a fan of the “Base” name, but I’m not sure what to call it.